lib/src/components/NxToast/__tests__/NxToast.test.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {
+  faCheckCircle,
+  faExclamationCircle,
+  faExclamationTriangle,
+  faInfoCircle
+} from '@fortawesome/free-solid-svg-icons';
+import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import NxCloseButton from '../../NxCloseButton/NxCloseButton';
+import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+import NxToast from '../NxToast';
+import { NxToastProps } from '../types';
+
+describe('NxToast', function() {
+  const simpleProps: NxToastProps = {
+        toastId: 1,
+        type: 'success',
+        message: 'Test toast message',
+        toastContainerRef: null
+      },
+      getShallow = getShallowComponent<NxToastProps>(NxToast, simpleProps);
```

Comment:
How much would you hate the idea that at this point, these should be RTL tests

Reply:
> How much would you hate the idea that at this point, these should be RTL tests

Sorry, these are outdated and written by Abhinav prior to us starting to use RTL. I deleted this in [b8d9ed5](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/b8d9ed50904a6e7267009970919e675c751a7b6a), but somehow brought it back in (although can't yet find in what commit). Sorry you had to go through the tests again. I'll remove them again.

Reply:
> Sorry you had to go through the tests again

Don't worry, I didn't really